docs/articles/documentation/reference/api/runner/filter.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title: Runner.filter Method
+permalink: /documentation/reference/api/runner/filter.html
+---
+# Runner.filter Method
+
+Allows you to select which tests should be run.
+
+```text
+filter(callback) → this
+```
+
+Parameter  | Type                                                                  | Description
+---------- | --------------------------------------------------------------------- | ----------------------------------------------------------------
+`callback` | `function(testName, fixtureName, fixturePath, testMeta, fixtureMeta)` | The callback that determines if a particular test should be run.
+
+The callback function is called for each test in the files the [src](src.md) method specifies.
```

Comment:
Maybe we can split this into two sentences? This one does not seem clear enough for the docs